docs/docs/frigate/installation.md

```diff
@@ -78,6 +81,12 @@ $ python -c 'print("{:.2f}MB".format(((1280 * 720 * 1.5 * 9 + 270480) / 1048576)
 
 The shm size cannot be set per container for Home Assistant add-ons. However, this is probably not required since by default Home Assistant Supervisor allocates `/dev/shm` with half the size of your total memory. If your machine has 8GB of memory, chances are that Frigate will have access to up to 4GB without any additional configuration.
 
+:::warning
```

Comment:
This seems redundant to me given the first addition

Reply:
My thinking was it's easy to gloss over those sections and doesn't hurt to have a warning banner? If disagree, happy to remove 👍

Reply:
Right. Let's make the first one a warning instead of a header section and remove the second.

Reply:
Sounds good 👍